specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-30/disk.json

```diff
@@ -2089,6 +2119,11 @@
           "type": "integer",
           "format": "int64",
           "description": "If createOption is Upload, this is the size of the contents of the upload including the VHD footer. This value should be between 20972032 (20 MiB + 512 bytes for the VHD footer) and 35183298347520 bytes (32 TiB + 512 bytes for the VHD footer)."
+        },
+        "logicalSectorSize": {
```

Comment:
please add example where logicalSectorSize is present in Get and also example where customer should be able to update an existing disk with different logicalSectorSize

Reply:
Added example which create an ultra disk with logicalSectorSize. Customer cannot update logicalSectorSize with different value.